src/AccessibilityInsights.SharedUxTests/Telemetry/LoggerUnitTests.cs

```diff
@@ -1,60 +1,64 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using AccessibilityInsights.SharedUx.Telemetry;
-using AccessibilityInsights.Extensions.Interfaces.Telemetry;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System;
 using System.Collections.Generic;
-#if FAKES_SUPPORTED
-using AccessibilityInsights.SharedUx.Telemetry.Fakes;
-using AccessibilityInsights.Extensions.Interfaces.Telemetry.Fakes;
-using Microsoft.QualityTools.Testing.Fakes;
-#endif
+using Moq;
+
+using TelemetryPropertyBag = System.Collections.Generic.IReadOnlyDictionary<AccessibilityInsights.SharedUx.Telemetry.TelemetryProperty, string>;
+using StringPropertyBag = System.Collections.Generic.IReadOnlyDictionary<string, string>;
 
 namespace AccessibilityInsights.SharedUXTests.Telemetry
 {
     [TestClass]
     public class LoggerUnitTests
     {
-#if FAKES_SUPPORTED
-        [TestMethod]
-        [Timeout(1000)]
-        public void IsEnabled_ImplementationIsNull_ReturnsFalse()
+        const TelemetryAction Action1 = (TelemetryAction)1;
+        const TelemetryAction Action2 = (TelemetryAction)2;
+        const TelemetryProperty Property1 = (TelemetryProperty)1;
+        const TelemetryProperty Property2 = (TelemetryProperty)2;
+        const TelemetryProperty Property3 = (TelemetryProperty)3;
+        const string Value1 = "January";
+        const string Value2 = "February";
+        const string Value3 = "March";
+
+        private Mock<ITelemetrySink> _sinkMock;
+
+        [TestInitialize]
+        public void BeforeEachTest()
         {
-            using (ShimsContext.Create())
-            {
-                ShimTelemetrySink.TelemetryGet = () => null;
+            _sinkMock = new Mock<ITelemetrySink>(MockBehavior.Strict);
+            Logger.SetTelemetrySink(_sinkMock.Object);
+        }
 
-                Assert.IsFalse(Logger.IsEnabled);
-            }
+        [TestCleanup]
+        public void AfterEachTest()
+        {
+            Logger.SetTelemetrySink(null);  // Reset to default sink
         }
 
         [TestMethod]
-        [Timeout(1000)]
-        public void IsEnabled_ImplementationIsNotNull_IsTelemetryAllowedGetIsTrue_ReturnsTrue()
+        [Timeout(2000)]
```

Comment:
intentional? It's identical to the method below, so it seems the timeouts should be similar